third_party/build/bin/aioh2-build.sh

```diff
@@ -11,37 +11,68 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-
 #build package from aioh2 master branch.
 
+set -e
+SCRIPT_DIR="$(dirname "$(realpath "$0")")"
+source "$SCRIPT_DIR"/../lib/util.sh
 PKGNAME=aioh2
 WORK_DIR=/tmp/build-"$PKGNAME"
-
 PWD1=$(pwd)
 
+function buildrequires() {
+    echo \
+        python3-venv \
+        python3-all \
+        debhelper \
+        dh-python \
+        python3-stem \
+        python3-stdeb \
+        python3-h2 \
+        python3-setuptools \
+        python3-distutils \
+        fakeroot
+
+}
+
+if_subcommand_exec
+
+sudo apt-get -y install \
```

Comment:
I wonder why we need `apt-get install` here. Doesn't `build.py` parse `buildrequires` and install the required packages already?